fe/index.html

```diff
@@ -2,16 +2,24 @@
 <html>
     <head>
         <title>おはトリ</title>
+        <meta charset="utf-8" />
```

Comment:
日本語の文字が出てくる前の部分で charset を指定してください
```suggestion
        <meta charset="utf-8" />
        <title>おはトリ</title>
```